apps/addons/management/commands/approve_addons.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+from django.core.management.base import BaseCommand, CommandError
+
+import amo
+from addons.models import Addon
+from editors.helpers import ReviewHelper
+
+
+class Command(BaseCommand):
+    args = '<addon_guid addon_guid ...>'
+    help = 'Approve a list of add-ons, given their GUIDs.'
+
+    def handle(self, *args, **options):
+        if len(args) == 0:  # No GUID provided?
+            raise CommandError(
+                'Please provide at least one add-on guid to approve.')
+
```

Comment:
We could indeed add pagination, or decide that "meh, it's a 'one time script', that can be called several times instead of once if needed with manual chunking". What do you think?